projects/whydense/cifar/mobilenet_cifar.py

```diff
@@ -0,0 +1,190 @@
+#  Numenta Platform for Intelligent Computing (NuPIC)
```

Comment:
Shouldn't this be in frameworks/pytorch?

Reply:
I dont know. It seems to specialized to be used by some other project.